pattern-1/README.md

```diff
@@ -1,141 +1,121 @@
-
-    ```
-    https://wso2ei.sys.<domain_name>/carbon
-    ```
-
-## Delete deployment
-
-1. Delete the deployment.
-
-    ```
-    bosh -e <environment-alias> -d <name> delete-deployment
-    ```
-
-2. **[Optional]** Cleanup the BOSH release, stemcell, disks and etc.
-
-    ```
-    bosh -e <environment-alias> clean-up --all
-    ```
-
-## BOSH release structure
-
-Structure of the directories and files of the BOSH release is as follows:
-
-```
-└── bosh-release
-    ├── config
-    ├── deployment
-    ├── dist
-    ├── images
-    ├── jobs
-    ├── packages
-    ├── src
-    └── create.sh
-```
-
-## References
-
-* [BOSH CLI v2 commands](https://bosh.io/docs/cli-v2.html)
+    The build process takes up to 5 minutes. After the build is complete, a tile named `wso2ei-tile-0.0.1.pivotal` is created in the root of the `/tile/product` directory.
+    
+## Install Enterprise Integrator in PCF
+1. Log in to PCF Ops Manager and upload the tile built by clicking **Import a Product**.
+2. After the tile is uploaded, add the tile to the PCF environment by clicking the + icon next to it.
+3. After the tile is added to the environment, click on the **Enterprise Integrator** tile in the PCF environment to add configurations to the setup.
+    ![Enterprise-Integrator](images/new-tile.png "Enterprise Integrator Tile")
+4. Set up the Enterprise Integrator tile.
+    1. AZ and Network Assignments Page:
+        ![az-network-assignments](images/az-assignment.png "AZs and Network Assignments")
+        * Place singleton jobs in: Select the AZ in which the Enterprise Integrator VM needs to run. The broker runs as a singleton job
+        * Balance other jobs in: Select any combination of AZs.
+        * Network: Select pcf-pas-network
+        
+        Click Save.
+    2. Registry Database Connection Information:
+        * **JDBC URL**:
+            * **MySQL**: `jdbc:mysql://<hostname>:<port>/<db_name>?autoReconnect=true&amp;useSSL=false`
+            * **MS SQL**: `jdbc:sqlserver://<hostname>:<port>;databaseName=<db_name>;`
+        * **Driver Class Name**: Select the class name of the JDBC driver relevant to the database being used.
+        * **Validation Query**: `SELECT 1`
+        * **Username**: Username for database
+        * **Password**: Password for database
+        
+        Click Save.
+    3. User Management Database Connection Information:
+        * **JDBC URL**:
+            * **MySQL**: `jdbc:mysql://<hostname>:<port>/<db_name>?autoReconnect=true&amp;useSSL=false`
+            * **MS SQL**: `jdbc:sqlserver://<hostname>:<port>;databaseName=<db_name>;`
+        * **Driver Class Name**: Select the class name of the JDBC driver relevant to the database being used.
+        * **Validation Query**: `SELECT 1`
+        * **Username**: Username for database
+        * **Password**: Password for database
+            
+            Click Save.
+    4. Enterprise Integrator - Analytics Clustering Database connection information
```

Comment:
For Integrator pattern-1, do we need the Analytics database configuration information?